subsys/bootloader/Kconfig

```diff
@@ -121,8 +121,8 @@ menuconfig IS_SECURE_BOOTLOADER
 
 if IS_SECURE_BOOTLOADER
 
-partition=B0
-partition-size=0x8000
+partition=B0_IMAGE
```

Comment:
should this be lower case? I see now that that is the case in the other Kconfig files.

Reply:
IN MCUboot they are upper case, so I think this line is currently following convention. We can change it later.

Reply:
`B0_image` middle ground?

Reply:
I think we should leave it as is for now.

Reply:
I agree that we follow what is there, just me mis-remembering :)